src/Sulu/Bundle/TranslateBundle/Command/ExportCommand.php

```diff
@@ -76,50 +64,93 @@ protected function configure()
                 'sets the filename of the exported file',
                 'sulu'
             );
+
+        $this->setDescription(
+            'Loads translation messages as defined by the parameters '
+            . 'from the bundles and writes them to a file, which also can be configured '
+            . 'via the parameters. If a locale got specified, only the translation '
+            . 'corresponding to that locale get exported. Otherwise the translation '
+            . 'files get exported for all locales defined in the system.'
```

Comment:
Use [NowDoc](http://php.net/manual/de/language.types.string.php#language.types.string.syntax.nowdoc) for the description.